parm/config/config.base.emc.dyn

```diff
@@ -29,6 +29,10 @@ export UTILgfs=$HOMEgfs/util
 export EXECgfs=$HOMEgfs/exec
 export SCRgfs=$HOMEgfs/scripts
 
+export FIXcice=$HOMEgfs/fix/fix_cice
+export FIXmom=$HOMEgfs/fix/fix_mom6
+export OCNFIXDIR=$HOMEgfs/fix/fix_reg2grb2
```

Comment:
Fine for now, but is there a reason why this is not `FIXreg2grb2` or atleast `FIXocn`

Reply:
Took as-is from feature/coupled-crow example configs, happy to change if others agree. I assume changes will need to be made at the script level too if so. @WalterKolczynski-NOAA @JessicaMeixner-NOAA thoughts?

Reply:
We should add this to a list somewhere to get fixed. No reason this can't or shouldn't be changed/cleaned up.

Reply:
Yeah, along with the DO_* and a bunch of other settings, everything needs a few consistency updates.

Reply:
Replacing `OCNFIXDIR` with `FIXreg2grb2`.

Reply:
If you replace that, are you also replacing it where the old variable is used in scripts?

Reply:
Yes, that was something we discussed in the tag-up this morning. The same variable in the scripts will also get updated when we get to that part.